### DISCUSSION TITLE
Run storybook, but just to warm up the cache?

### DISCUSSION THREAD

**Discussion by kaelig** [storybookjs/storybook#13737] [Help] [answered]
👋 I'm setting up snapshots of "pre-bootstrapped" Docker container images (with all dependencies pre-installed), so developers can quickly start storybook in a cloud development environment (think: GitHub Codespaces).

At the moment, when devs open the workspace, and they run `yarn storybook`, it takes ~1m until Storybook shows up the browser.
When running the same command a second time, it takes 30s until Storybook shows up the browser. I'd love to see if I can always make it this fast for them (30s rather than 1m).

- I understand that the second run is faster thanks to a warm cache, is that a correct assumption?
- If so, what's the best way to warm up the cache without starting the server?

I tried with `yarn run storybook --smoke-test`, but the `yarn storybook` run that follows takes as long as it did with a cold cache.

**Comment by ghengeveld** [comment]
The 2nd run is faster because of the manager cache, i.e. the JS bundle for the manager won't get recompiled the second time. This cache is located in `node_modules/.cache/storybook`, so you could probably ship it along with your container image.

**Comment by kaelig** [comment, marked as answer]
In case anybody else has the same use-case, warming up the cache worked for me in v6.2.0-alpha.24 using this trick provided by @ghengeveld:

```
yarn storybook --smoke-test --preview-url=""
```

**Reply by kaelig** [reply]
This trick doesn't seem to work anymore (or at least it doesn't work on a codebase I'm currently working on).

I always get this:

```
info => Ignoring cached manager due to change in manager config
```

Any pointers welcome!
